tracer/src/Datadog.Trace/LibDatadog/TraceExporterErrorHandle.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="TraceExporterErrorHandle.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.LibDatadog;
+
+internal class TraceExporterErrorHandle : SafeHandle
```

Comment:
Should this derive from `ErrorHandle`? Or should `ErrorHandle` actually be `LoggerErrorHandle`?